calendarpal/src/main/java/org/dmfs/android/calendarpal/operations/EventRelated.java

```diff
@@ -35,30 +31,12 @@
  *
  * @author Marten Gajda
  */
-public final class EventRelated<T> implements InsertOperation<T>
+public final class EventRelated<T> extends DelegatingOperation<T>
```

Comment:
`EventRelated` should stay an `InsertOperation` even though the documentation doesn't explicitly forbid it, you really should not change the `event_id` of an attendee, reminder or extended data entry.

Reply:
Reverted that change. Let me know if I should rebase and squash.